tests/TestHarness/launch_transaction_generators.py

```diff
@@ -91,7 +91,7 @@ def launch(self, waitToComplete=True):
                         '--actions-data', f'{self.actionsData}',
                         '--actions-auths', f'{self.actionsAuths}',
                         '--peer-endpoint', f'{self.peerEndpoint}',
-                        '--port', f'{self.port}'
+                        '--port', f'{self.port}',
```

Comment:
Why?

Reply:
If you remove the added space your PR will not have touched `launch_transaction_generators.py`.  Not sure it is worth adding a space when you could have a clean diff here and the new space isn't really doing anything.